src/Package/Impl/DataInspect/REnvironmentProvider.cs

```diff
@@ -39,8 +39,16 @@ internal class REnvironmentProvider : IREnvironmentProvider, IDisposable {
 
             Debug.Assert(result.JsonResult != null);
 
+            var envs = new List<REnvironment>();
+            var jarray = result.JsonResult as JArray;
+            if (jarray != null) {
```

Comment:
This should try to validate and throw if received JSON is not what is expected, rather than silently producing an empty list.